### PR TITLE
ImagePreview closes GoalPreview on ESC

### PR DESCRIPTION
**Pull request by Troff8**
## PR includes

- [x] Bug Fix

## Related issues

Resolve [#1507 ](https://github.com/taskany-inc/issues/issues/1507)

### CODE REVIEW

**Review by 9teen90nine — APPROVED** [review]

File: src/components/ImageFullScreen.tsx [hunk 1]
```diff
@@ -15,11 +15,26 @@ const StyledImage = styled.img`
...
+    useClickOutside(ref, () => {
+        dispatchModalEvent(ModalEvent.ImageFullScreen)();
+    });
+
+    const [onESC] = useKeyboard([KeyCode.Escape], () => {
+        dispatchModalEvent(ModalEvent.ImageFullScreen)();
+    });
+
+    useEffect(() => {
+        setTimeout(() => ref.current && ref.current.focus(), 0);
```
Comment (awinogradov) [inline src/components/ImageFullScreen.tsx:32]: `ref.current?.focus()` ?

**Review by awinogradov — CHANGES_REQUESTED** [review]
image

You have commits with a non github account

File: src/components/ImageFullScreen.tsx (hunk 1 above)
Comment (Troff8) [reply src/components/ImageFullScreen.tsx:32]: tried, doesn't work, had to use setTimeout

**Comment by awinogradov** [comment]
Also, in description of PR you set feature but this is bug fix. Same in your commit message.

**Review by awinogradov — APPROVED** [review]

**Pull request merged**